sanic/request.py

```diff
@@ -142,7 +142,7 @@ def raw_args(self):
     @property
     def cookies(self):
         if self._cookies is None:
-            cookie = self.headers.get('Cookie') or self.headers.get('cookie')
+            cookie = self.headers.get('Cookie')
```

Comment:
it's not related. this is code cleanup, since `header` is a Case Insensitive Dict.